openai/requirements.txt

```diff
@@ -0,0 +1 @@
+openai
```

Comment:
Ideally to pin the version to the current major (interval) in case of the upstream breaking changes